CHANGELOG.md

```diff
@@ -1,14 +1,404 @@
 # Changelog
 
-Note: The changelog is severely out of date, and will be updated *later*.
-
-These changes have been made since the current 2.2.0 release. If you build the project as-is, it will include all of them.
 
 > What's the point of going through all the trouble of the debug process if you're going to go and fiddle with the game afterwards…?
 >
 > — Satoru Iwata, "[Iwata Asks – Pokémon HeartGold Version & SoulSilver Version](https://www.nintendo.co.uk/Iwata-Asks/Iwata-Asks-Pokemon-HeartGold-Version-SoulSilver-Version/Iwata-Asks-Pokemon-HeartGold-Version-SoulSilver-Version/1-Just-Making-The-Last-Train/1-Just-Making-The-Last-Train-225842.html)"
 
 
+Polished Crystal v3.1.1 brings with it a lot of changes. What started as a Dex expansion, through three years of development it has evolved into a strikingly different game from the v3.0.0-beta release with numerous added features from newer generations, a plethora of code optimizations, and many quality of life improvements. Ultimately, the game aims to feel like a modern Pokemon game with retro graphics. The rule of thumb is to assume that things work like modern games, and if it doesn't, it is either a bug or an oversight.
```

Comment:
None of these paragraphs are necessary. Just keep the quote then get into the changelog. This sort of summarizing belongs in the README or FAQ.